pkg/kubectl/cmd/taint.go

```diff
@@ -113,42 +120,56 @@ func NewCmdTaint(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 // reorganizeTaints returns the updated set of taints, taking into account old taints that were not updated,
 // old taints that were updated, old taints that were deleted, and new taints.
-func reorganizeTaints(obj runtime.Object, overwrite bool, taintsToAdd []v1.Taint, taintsToRemove []v1.Taint) ([]v1.Taint, error) {
-	node, ok := obj.(*v1.Node)
-	if !ok {
-		return nil, fmt.Errorf("unexpected type %T, expected Node", obj)
-	}
-
+func reorganizeTaints(node *v1.Node, overwrite bool, taintsToAdd []v1.Taint, taintsToRemove []v1.Taint) (string, []v1.Taint, error) {
```

Comment:
You mean somewhere in one of scheduler's packages or api packages? Most of the functions for getting taints are in api/helpers.go. If you want I can make that change but I guess that pre-dates my changes.